src/algebra/polynomial/big_operators.lean

```diff
@@ -138,6 +228,11 @@ lemma coeff_zero_prod :
   (∏ i in s, f i).coeff 0 = ∏ i in s, (f i).coeff 0 :=
 by simpa using coeff_zero_multiset_prod (s.1.map f)
 
+lemma coeff_finset_sum (f : ι → polynomial R) (n : ℕ) :
+  coeff (s.sum f) n = s.sum (λ p, coeff (f p) n) :=
+let g : polynomial R →+ R := add_monoid_hom.mk (λ p, coeff p n) (by simp) (by simp) in
```

Comment:
This should probably exist somewhere already. If it really doesn't, `coeff_add_monoid_hom` would be fine as a name.

Reply:
ah, we have `lcoeff`.